.github/ansible/staging.eu-west-1.hosts.yaml

```diff
@@ -8,6 +8,14 @@ storage:
       pg_distrib_dir: /usr/local
       metric_collection_endpoint: http://neon-internal-api.aws.neon.build/billing/api/v1/usage_events
       metric_collection_interval: 10min
+      disk_usage_based_eviction:
+        max_usage_pct: 80
+        # TODO: learn typical resident-size growth rate [GiB/minute] and configure
+        # min_avail_bytes such that we have X minutes of headroom.
```

Comment:
I would say hours instead of minutes :)

Reply:
I don't think hours will be practically possible. For example, with my layer scrape tenant, we see a growth rate of 1 GiB/min when it executes queries that need on-demand downloads for everything.